js/todo.js

```diff
@@ -36,18 +40,22 @@ let createTodo = event => {
     todoText.classList.remove("error")
     
     createTodoElement(elementId, todoText.value);
-    storage.setItem(elementId, todoText.value);
+    storage.set({
+      [elementId]: todoText.value
+    });
 
     // TODO: Rework this part
     // makes the new todo also follow the contrast
-    if (storage.getItem("backgroundColor") !== null) {
-      let backgroundColor = document.body.style.backgroundColor;
-      backgroundColor = rgbList(backgroundColor);
-      for(let i = 0; i < 3; i++) {
-        backgroundColor[i] = parseInt(backgroundColor[i]);
+    storage.get("backgroundColor", object => {
+      if (object.backgroundColor !== null) {
+        let backgroundColor = document.body.style.backgroundColor;
+        backgroundColor = rgbList(backgroundColor);
+        for (let i = 0; i < 3; i++) {
+          backgroundColor[i] = parseInt(backgroundColor[i]);
+        }
+        fixContrast(backgroundColor);
```

Comment:
Something is funky when using the default theme, when creating todos it will change the color to white. Like this;
![bug](https://user-images.githubusercontent.com/47867973/123649415-03dd6880-d82a-11eb-8b2e-c12ed41effc5.png)